### PR TITLE
video_minimum_maximum: Implement src operand selectors

### PR DESCRIPTION
**Pull request by ameerj**
Validated with unit tests. 

- Used by Pokemon Legends: Arceus.

### CODE REVIEW

File: src/shader_recompiler/frontend/maxwell/translate/impl/video_minimum_maximum.cpp [hunk 1]
```diff
@@ -57,16 +57,6 @@ void TranslatorVisitor::VMNMX(u64 insn) {
...
-    // Selectors were shown to default to 2 in unit tests
-    if (vmnmx.src_a_selector != 2) {
-        throw NotImplementedException("VMNMX Selector {}", vmnmx.src_a_selector.Value());
-    }
-    if (vmnmx.src_b_selector != 2) {
-        throw NotImplementedException("VMNMX Selector {}", vmnmx.src_b_selector.Value());
-    }
-    if (vmnmx.src_a_width != VideoWidth::Word) {
-        throw NotImplementedException("VMNMX Source Width {}", vmnmx.src_a_width.Value());
-    }
```
Comment (FernandoS27) [inline src/shader_recompiler/frontend/maxwell/translate/impl/video_minimum_maximum.cpp:69]: Why was this removed?

Comment (ameerj) [reply src/shader_recompiler/frontend/maxwell/translate/impl/video_minimum_maximum.cpp:69]: It's implemented with the selectors. Verified here: https://github.com/ReinUsesLisp/nxgpucatch/commit/ca83ec81f7e41dc1e8a228d8bff84db090dda6a8

**Review by Morph1984 — APPROVED** [review]
